### ISSUE TITLE
Update Travis CI and Coverity configuration

### ISSUE THREAD

**Issue by anonimal** [monero-project/kovri#20]
Currently both point to anonimal/i2pd. Once @fluffypony is done working on his part, we can edit the config and complete this part of the transition.

**Comment by anonimal** [comment]
From paste:
```
- Github->Integrations->Travis CI then follow directions. Apply new credentials to .travis.yml in branch 'kovri' or give me credentials and I'll push.
- https://scan.coverity.com/travis_ci - then apply updated credentials to .travis.yml in branch 'coverity_scan' or give me credentials and I'll push.
- Create "kovri-ci" or "kovri" account on respective IRC channels
- Encrypt its password for coverage
  $ gem install travis -v 1.8.0 --no-rdoc --no-ri
  $ travis encrypt 'chat.freenode.net#kovri-dev' --org -r monero-project/
- Add encrypted password to .travis.yml in branch 'kovri' or 'coverity_scan' or I can push to both.
```

**Comment by anonimal** [comment]
```
$ travis encrypt 'chat.freenode.net#kovri-dev' --org -r monero-project/
repository not known to https://api.travis-ci.org/: monero-project/
```
@fluffypony I thought kovri was setup yesterday?

**Comment by anonimal** [comment]
Ping @fluffypony

**Comment by anonimal** [comment]
All that's left for this ticket is Coverity coverage.

**Comment by anonimal** [comment]
@fluffypony can you paste the problem/error messages you're receiving when trying to register with coverity? I could not register monero-project/kovri as it was "Already registered". Registering anonimal/kovri would not give us integrated travis/coverity coverage because travis is registered with monero-project/kovri.  Re-registering travis with anonimal/kovri and subsequently using that instead of monero-project/kovri it is a terrible idea so I don't want to do it. In other words, we need to get in coverity's face to resolve this issue since they still haven't responded to your emails (it's been many weeks now).

So far I've had no problems with the registration. Maybe when you paste the error messages I can be more helpful.

**Comment by fluffypony** [comment]
@anonimal ok, here goes.

![Step 1](http://i.imgur.com/Bxank8C.png)

I start by signing in to Coverity, and going to the Kovri project.

![Step 2](http://i.imgur.com/ZMwsJoi.png)

I obviously want to get our build going, so I click the "Submit a Build" link.

![Step 3](http://i.imgur.com/OGWNjD3.png)

Instead of doing the build upload, I want to "Configure Travis CI", so I click there.

At this point I have to wait several minutes for the next page to load. Must easily be 5 minutes. I'm met with this error:

![Step 4](http://i.imgur.com/fOFyoao.png)

SO, I switch to Travis CI and see if I've done something wrong there.

![Step 5](http://i.imgur.com/SaL4MAV.png)

Nope, looks good, and it's ticked. But I click the little "gear" icon just in case.

![Step 6](http://i.imgur.com/vjAhEXu.png)

Looks fine here too, all synced up and working. Maybe under More Options -> Settings there's some hidden option.

![Step 7](http://i.imgur.com/WjZ9f4e.png)

Aaaand nothing. No hidden option to enable sync or do something funny. Everything is configured as expected, but Coverity just refuses to pick it up as such.

**Comment by anonimal** [comment]
fluffypony: the screenshot king.

The only response that I can come up with now is to email them again but this time sending a link to your [comment with screenshots](https://github.com/monero-project/kovri/issues/20#issuecomment-185157648). They also have [hotlines](https://www.coverity.com/services/technical-support/) available.  If they don't respond within 48 hours, please send me the email address you are trying to contact and I'll send them a message too. I would ping them here if they made themselves available on github but alas, they don't (AFAICT).

**Comment by anonimal** [comment]
@fluffypony status update?

**Comment by anonimal** [comment]
```
2016-05-22  &anonimal Onward to #20.
2016-05-22  meeting-bot [fluffypony] 20 - I can try Coverity again, let's see if it gives me the same issue
2016-05-22  meeting-bot [fluffypony] I've not heard from them despite opening tickets etc.
2016-05-22  &anonimal Links? Maybe I can comment/ping them too.
2016-05-22  meeting-bot [fluffypony] anonimal: screenshots and process I followed is in that ticket
2016-05-22  meeting-bot [fluffypony] issue I mean
2016-05-22  &anonimal Ok, well, it may come down to a phone call or two.
2016-05-22  &anonimal Is that something you'd be willing to do?
2016-05-22  &anonimal If not, we can skip the integration and try to do it manually.
2016-05-22  meeting-bot [fluffypony] sure
2016-05-22  meeting-bot [fluffypony] one second just checking coverity
2016-05-22  meeting-bot [fluffypony] oooooh it works
2016-05-22  meeting-bot [fluffypony] they must've fixed it and just not let us know
2016-05-22  &anonimal Fantastic!
2016-05-22  meeting-bot [fluffypony] anonimal: I'll PM you with coverity details
```

**Comment by anonimal** [comment]
Resolved with #180
